main/java/BeforeAfter.java

```diff
@@ -0,0 +1,69 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import shop.Cart;
+import parser.JsonParser;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.File;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class BeforeAfter {
```

Comment:
I recommend remove separate class for before/after hooks. Store hooks in classes with tests and add them to classes where it is really needed

Reply:
I thought that the sentence "Usage of Before/After annotations is required" in Task 10 means that I must use before/ After for every single test.
Ok, I will move Before/ After from separate class and add in test cases when needed.

Reply:
Correction is applied.